src/test/resources/fixtures/hmh/hmh-ri/released/caliperEventAssessmentStarted.json

```diff
@@ -1,35 +1,49 @@
 {
-  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
-  "type":"AssessmentEvent",
-  "id":"urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
-  "actor":{
-    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
-    "type":"Person"
-  },
-  "action":"Started",
-  "object":{
-    "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
-    "type":"Assessment",
-    "name":"ReadingInventory",
-    "maxAttempts":1,
-    "maxSubmits":1,
-    "maxScore":25.0,
-    "version":"1.0"
-  },
-  "eventTime":"2016-11-15T10:15:00.000Z",
-  "edApp": "https://www.hmhco.com/ReadingInventory",
-  "membership":{
-    "id":"urn:uuid:randomGuidId",
-    "type":"Membership",
-    "organization":{
-      "id":"urn:uuid:ab570d90-3791-4048-a5ee-759657ddccef",
-      "type":"Organization",
-      "subOrganizationOf":{
-        "id":"urn:uuid:ea827de8-0ce9-4fe4-a28c-eb388bd9eb92",
-        "type":"Organization"
-      }
-    },
-    "roles":[ "Learner" ],
-    "status": "Active"
-  }
+	"@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+	"type": "AssessmentEvent",
+	"id": "urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
+	"actor": {
+		"id": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+		"type": "Person"
+	},
+	"action": "Started",
+	"object": {
+		"id": "urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+		"type": "Assessment",
+		"name": "ReadingInventory",
+		"maxAttempts": 1,
+		"maxSubmits": 1,
+		"maxScore": 25.0,
+		"version": "1.0"
+	},
+	"eventTime": "2016-11-15T10:15:00.000Z",
+	"edApp": {
```

Comment:
Ed Analytics only supports edApp being a String so the following change will prevent this message from flowing through its pipeline. Is there perhaps an alternative location for this new data?

Reply:
I could put it under general extension but i thought we want to keep caliper entities to be consistent and attach them to the right parent entities. Here application and discipline code although in extensions felt right to keep it under the edapp as they are attributes of the app.